### DISCUSSION TITLE
How to get config in Hook?

### DISCUSSION THREAD

**Discussion by jiekun** [redis/go-redis#2113] [Q&A] [answered]
Hi. Basically we could use data inside `CMD` and `ctx` now. But could we open more info/data to the hook impl?

There is no way to report the redis address / port / name to metric collector like prometheus/grafana or opentracing.

**Comment by vmihailenco** [comment, marked as answer]
You can save some information on the struct that implements hooks interface - see https://github.com/go-redis/redis/blob/master/extra/redisotel/redisotel.go#L21-L22

**Reply by jiekun** [reply]
I see. Thank you for this example. Easy to understand
